gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/TileSourceEntity.java

```diff
@@ -117,6 +131,8 @@ public static Builder builder() {
 
     public abstract Builder setState(TileEntityState newState);
 
+    public abstract Builder setAreaCount(Integer areaCount);
```

Comment:
If area count can never be null please use `int` instead.

Reply:
048dd80